### DISCUSSION TITLE
Windows 11 -> symfony/orm-pack extension.

### DISCUSSION THREAD

**Discussion by Mariusz225** [dunglas/symfony-docker#415] [Q&A]
I have installed the symfony/orm-pack extension.

Everything works fine in Ubuntu. However, when I run this project on Windows 11, I encounter an issue where PHP keeps displaying the message: "Still waiting for db to be ready... Or maybe the db is not reachable. x attempts left."

The database container seems to be running based on what I see in Docker Desktop.

What can be a problem?

**Comment by maxhelias** [comment]
Hi, often this problem is related to the execution rights of the `bin/console` file, which are not allowed.
